cmd/launcher/launcher.go

```diff
@@ -381,6 +381,9 @@ func main() {
 		if opts.insecureTLS {
 			controlOpts = append(controlOpts, control.WithInsecureSkipVerify())
 		}
+		if opts.disableControlTLS {
+			controlOpts = append(controlOpts, control.WithDisableTLS())
```

Comment:
s/withDisableTLS/withDisableControlTLS

Reply:
That's in the `control` package. Thoughts to keep it as is?

Reply:
oh now i see, nvm 👍